prov/efa/src/rxr/rxr_msg.c

```diff
@@ -224,6 +224,30 @@ ssize_t rxr_msg_post_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry)
 					 ctrl_type + tagged, 0, 0);
 	}
 
+	/*
+	 * Force the LONGREAD protocol for Neuron buffers, regardless of what
+	 * is specified by the user for protocol switch over points.
+	 */
+	if (efa_ep_is_neuron_mr(tx_entry->desc[0])) {
```

Comment:
So we still use eager message protocol for neuron memory buffer < 8K?

Reply:
Yes, that's right. I suspect this may change in the future, but for now the neuron team requested we only issue RDMA reads for long messages.